content/applications/sales/sales/products_prices/loyalty_discount.rst

```diff
@@ -0,0 +1,188 @@
+to add *conditions* to the program. This reveals a :guilabel:`Create Conditional rules` pop-up
+window.
+
+.. image:: loyalty_discount/price-conditional-rewards.png
+   :align: center
+   :alt: Conditional rewards
+
+.. note::
+   The options for :guilabel:`Conditional rules` vary depending on the selected :ref:`Program Type
+   <pricing_management/program-types>`.
+
+The following options are available for configuring conditional rules:
+
+- :guilabel:`Discount Code`: Enter a custom code to be used for the :guilabel:`Discount Code`
+  program, or use the default one generated by Odoo. This field is only available when the
+  :guilabel:`Program Type` is set to :guilabel:`Discount Code`.
+- :guilabel:`Minimum Quantity`: Enter the minimum number of products that must be purchased in order
+  to access the reward. Set the minimum quantity to at least `1` to ensure that the customer must
+  make a purchase in order to access the reward.
+- :guilabel:`Minimum Purchase`: Enter the minimum amount (in currency), with :guilabel:`tax
+  Included` or :guilabel:`tax Excluded`, that must be spent in order to access the reward. If both a
+  minimum quantity *and* minimum purchase amount are entered, then the customer's order must meet
+  both conditions.
+- :guilabel:`Products`: Select the specific product(s) for which the program applies. Leave this
+  field blank to apply it to all products.
+- :guilabel:`Categories`: Select the category of products for which the program applies. Choose
+  :guilabel:`All` to apply it to all product categories.
+- :guilabel:`Product Tag:` Select a tag to apply the program to products with that specific tag.
+- :guilabel:`Grant`: Enter the number of points the customer earns :guilabel:`per order`,
+  :guilabel:`per currency spent`, or :guilabel:`per unit paid` (for the :guilabel:`Loyalty Cards`
+  and :guilabel:`Buy X Get Y` programs).
+
+.. image:: loyalty_discount/price-conditions.png
+   :align: center
+   :alt: Conditional rules to get rewards through a program type
+
+Click :guilabel:`Save & Close` to save the rule and close the pop-up window, or click
+:guilabel:`Save & New` to save the rule and immediately create a new one.
+
+Rewards
+-------
+
+In the :guilabel:`Rules & Rewards` tab of the program form, click :guilabel:`Add` next to
+:guilabel:`Rewards` to add *rewards* to the program. This reveals a :guilabel:`Create Rewards`
+pop-up window.
+
+.. note::
+   The options for :guilabel:`Rewards` vary depending on the selected :ref:`Program Type
+   <pricing_management/program-types>`.
+
+The following options are available for configuring rewards:
+
+- :guilabel:`Reward Type`: Select the reward type among :guilabel:`Free Product`,
+  :guilabel:`Discount`, and :guilabel:`Free Shipping`. The other options for reward configuration
+  depend on the :guilabel:`Reward Type` selected.
+
+  - :guilabel:`Free Product` Reward Type:
+
```

Comment:
I think reiterating for each bullet of the reward type's as "X Reward Type" is redundant, when it is already implied by the parent bullet. 

I suggest formatting the reward type bullets as just "X" 

```suggestion
  - :guilabel:`Free Product`:

```